trigger_rollback.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2016
+set -eu
+
+repo="$1"
+user=greenpeace
+tag="$2"
+
+json=$(jq -n \
+  --arg VAL "$tag" \
+  --arg RELEASE_STAGE "rollback" \
+'{
+  "tag": $VAL,
+  "parameters": {
+    "build_num": $BUILD_NUM,
+    "release_stage": $RELEASE_STAGE
+  }
+}')
+
+echo "Build: ${user}/${repo}@${tag}"
```

Comment:
We echo here that it's triggering a rollback, currently the job output doesn't specify say it's triggering a rollback.

Reply:
Maybe we can echo the json of the request too?

Reply:
Good point. I added a couple of `echo` outputs there to make it more verbose.